#!/usr/bin/env python
PACKAGE='orbit_pantilt'
import roslib; roslib.load_manifest(PACKAGE)

from math import pi
from driver_base.msg import SensorLevels
from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

#gen.add("max_range", double_t, SensorLevels.RECONFIGURE_RUNNING,
#        "max_range corresponding to max sensor output",   5.0, 0.0, 20.0)

gen.add("pan_reset", bool_t, SensorLevels.RECONFIGURE_RUNNING,
        "reset pan position",  False)
gen.add("tilt_reset", bool_t, SensorLevels.RECONFIGURE_RUNNING,
        "reset tilt position",  False)

gen.add("pan", int_t, SensorLevels.RECONFIGURE_RUNNING,
        "pan position", 0, -94, 94)
gen.add("tilt", int_t, SensorLevels.RECONFIGURE_RUNNING,
        "tilt position", 0, -51, 51)

exit(gen.generate(PACKAGE, "orbit_pantilt", "OrbitPanTilt"))
